dao/src/main/java/org/thingsboard/server/dao/sql/query/DefaultAlarmQueryRepository.java

```diff
@@ -87,6 +88,13 @@ public class DefaultAlarmQueryRepository implements AlarmQueryRepository {
             " THEN (select name from entity_view where id = a.originator_id)" +
             " END, 'Deleted') as originator_name";
 
+    private static final String SELECT_ORIGINATOR_LABEL = " COALESCE(CASE" +
```

Comment:
Edge has a label as well as a name. Could you please add it to both SELECT_ORIGINATOR_NAME and SELECT_ORIGINATOR_LABEL statements?